measures/ApplyUpgrade/resources/constants.rb

```diff
@@ -28,6 +28,7 @@ def self.CostMultiplierChoices
       'Duct Unconditioned Surface Area (ft^2)',
       'Rim Joist Area, Above-Grade, Exterior (ft^2)',
       'Slab Perimeter, Exposed, Conditioned (ft)',
+      'Air Infiltration (ACH50)',
```

Comment:
There is already cost multiplier (`'Floor Area, Conditioned * Infiltration Reduction (ft^2 * Delta ACH50)'`) intended to be used for air sealing measures. How would you use the ACH50 directly for costing?

Reply:
This isn't permanent. It's the avenue I'm taking for reporting the ACH50 value (i.e., the HPXML file needs to be built and defaulted before I can get/calculate the ACH50 value).